pkg/build/nodeimage/buildcontext.go

```diff
@@ -135,6 +144,9 @@ func (c *buildContext) buildImage(bits kube.Bits) error {
 		"docker", "commit",
 		// we need to put this back after changing it when running the image
 		"--change", `ENTRYPOINT [ "/usr/local/bin/entrypoint", "/sbin/init" ]`,
+		// remove proxy settings since they're for the building process
+		// and should not be carried with the built image
+		"--change", `ENV HTTP_PROXY="" HTTPS_PROXY="" NO_PROXY=""`,
```

Comment:
I'm not familiar with this, what is this change required?

Reply:
Sorry I may not fully understand your question.  The `--change` option accepts a Dockerfile instruction and applies it to the outputted image. Detail is here: https://docs.docker.com/engine/reference/commandline/commit/#options

Reply:
but why do we need to use it?

Reply:
Right now it's only based on my local testing result (control panel won't start successfully if they're not removed), and I haven't figured out what's going on underneath. I'll offer a detailed explanation in a few days cause I can only do this after work, please bear with me.

Reply:
there is no rush, take your time, but if we add any option we need to be sure we don't break any existing workflow

Reply:
this is necessary so we don't wind up saving the env to the saved image